### PR TITLE
Update tables

### PR DESCRIPTION
**Pull request by NicolasAndrehh**
# Milestone - Update Tables
This pull request includes the following changes:

- Add new column species to animals table
- Insert new data into animals table
- First transaction result

> image

- Second and Third transaction result

> Transaction PostgreSQL

- Fourth transaction result

> Transaction2

- Answer questions using Queries: 

>``` 
> How many animals are there?
>```
> image

>```
> How many animals have never tried to escape?
>```
> image

>```
> What is the average weight of animals?
>```
> image

>```
> Who escapes the most, neutered or not neutered animals?
>```
> image

>```
> What is the minimum and maximum weight of each type of animal?
>```
> image

>```
> What is the average number of escape attempts per animal type of those born between 1990 and 2000?
>```
> image

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Nicolas, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly ensure that your ReadMe is professionally documented;  Add the _Future Features_ section to your ReadMe which should include 1-3 potential features to be added later.

- [NOT A REASON FOR INVALIDATION]
Also, The _acknowledgment_ section apparently contains unrelated content, I'll urge you to kindly add the appropriate content(s) under the _Acknowledgement_ section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by NicolasAndrehh** [comment]
Thanks for the feedback!

File: queries.sql [hunk 1]
```diff
@@ -14,4 +14,37 @@ SELECT * from animals WHERE neutered = TRUE;
 
 SELECT * from animals WHERE name != 'Gabumon';
 
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```
Comment (1GORDON) [inline queries.sql:18]: Great work @NicolasAndrehh 🤝

- [ ] You seem to have missed the first transaction. Kindly UPDATE the animals' table by setting the species column to unspecified, then roll back the change. You may check your project requirements for more clarity.

File: queries.sql [hunk 2]
```diff
@@ -14,4 +14,37 @@ SELECT * from animals WHERE neutered = TRUE;
...
+-- First Transaction 
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals; 
+
```
Comment (1GORDON) [inline queries.sql:32]: Great work @NicolasAndrehh 🤝

- [ ]  You seem to have missed another transaction. Kindly delete all records in the animals' table, then roll back the transaction. Kindly check your project requirements for more clarity.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @NicolasAndrehh ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NicolasAndrehh** [comment]
Thanks for the feedback!

**Review by Davidosky007 — APPROVED** [review]
Hi @NicolasAndrehh 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Transactions written properly 👍 
* Queries written properly 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**